script/dev-deploy

```diff
@@ -1026,6 +1029,136 @@ class NodeExporterService(Service):
         return cmds
 
 
+class SecurityService(Service):
+    KEYS = ["security"]
+
+    def __init__(self, cmdparsers):
+        self._cmdparsers = cmdparsers
+        self.cmd = self._cmdparsers.add_parser(
+            self.KEYS[0],
+            aliases=self.KEYS[1:],
+            help="Manage security service",
+        )
+
+    def init_parser(self):
+        action_subparser = self.cmd.add_subparsers(
+            title="action", dest="action_command"
+        )
+
+        # ------------------------------
+        # generate ca key
+        gen_ca_key_parser = argparse.ArgumentParser(add_help=False)
+        gen_ca_key_parser.add_argument(
+            "-o", "--output", default="ca.key.pem", action="store"
```

Comment:
the default action is already set to "store", no need to set it again.
https://docs.python.org/3/library/argparse.html#action